src/FixedDataTableRow.js

```diff
@@ -241,9 +248,11 @@ class FixedDataTableRowImpl extends React.Component {
       'public/fixedDataTableRow/fixedColumnsDivider': left > 0,
       'public/fixedDataTableRow/columnsShadow': this.props.scrollLeft > 0,
      });
+     // Subtract 1 for the border
```

Comment:
Why border is this? Can you make a CONST reference both places the border is defined?

Reply:
.fixedDataTableLayout/header border-bottom-width

Think it makes more to just modify the comment here to include that info, since it's only used here and defined in css.

Reply:
You can use BORDER_HEIGHT from FixedDataTable.js

Reply:
But that's for a different border, even though they have the same value. I can still define it and put a comment about where it comes from, but I was trying to say that I'm not sure it makes sense to define a const when it's only used in one place.

Reply:
Ah cool.  If it's philosophically different, then you're right, we should have a difference const.

I do think it makes sense to have a const though so someone reading the file in 6 months know that's what the 1 represents.  A comment can also serve this purpose like the [extra 18 here|https://github.com/schrodinger/fixed-data-table-2/blob/master/src/FixedDataTableRoot-test.js#L91].

For an example of challenges w/ leaving the magic number, consider the difficulty knowing what [30 * 100 - 300 is here|https://github.com/schrodinger/fixed-data-table-2/blob/master/src/FixedDataTableRoot-test.js#L138].

Reply:
Ok cool - is what I have from my existing follow up commit (8cf6d18) sufficient then?

https://github.com/schrodinger/fixed-data-table-2/pull/239/commits/8cf6d182e0d9803c8fe1ad8f14b2ffbfd45af815#r144370010